wgengine/netstack/netstack.go

```diff
@@ -55,6 +56,11 @@ type Impl struct {
 
 	mu  sync.Mutex
 	dns DNSMap
+	// connsOpenBySubnetIP keeps track of number of connections open
+	// for each subnet IP temporarily registered on netstack for active
+	// TCP connections, so they can be unregistered when connections are
+	// closed.
+	connsOpenBySubnetIP map[netaddr.IP]int
```

Comment:
there are fancier things we can do later for performance but this is nice and simple for now.